android_app/app/src/main/AndroidManifest.xml

```diff
@@ -9,6 +9,15 @@
     <uses-permission android:name="android.permission.RECEIVE_BOOT_COMPLETED"/>
     <uses-permission android:name="android.permission.ACCESS_COARSE_LOCATION"/>
 
+    <!-- Permission to allow read of data from the database through a ContentProvider.
+         Marked "dangerous" so that explicit user approval is required to read this data, not
+         just the permission implied from installing the app from the Play Store. -->
+    <permission
+        android:name="com.health.openscale.READ_DATA"
```

Comment:
Can this be just ".READ_DATA"? The dev version built by Travis adds ".dev" to the application id.